src/components/messages/multi-select-users.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+
+const MultiSelectUsers = ({users, checkHandler, uncheckHandler}) => {
```

Comment:
@galen94 @tomirisk You should be able to reuse this component for your use cases of multi-selecting users for creating a chat group and for creating a private story.

Reply:
Everything looks good. Thank you for creating the multi-select users component!